components/routes/Drawers.js

```diff
@@ -34,6 +35,15 @@ export const DefaultDrawer = () => {
               />
             )
           })} />
+          <Drawer.Screen name="Events Calendar" component={CalendarScreen}
+          options={({ navigation }) => ({
+            headerLeft: () => (
+              <HeaderBackButton
+                tintColor={'white'}
+                onPress={() => navigation.goBack()}
+              />
+            )
+          })} />
```

Comment:
We don't need this screen in the `Drawer` as it is already in the `Tab` navigation.

Reply:
Thx, Noted and applied.